packages/bot-web-ui/src/pages/tutorials/__tests__/tutorial-tab-desktop.spec.tsx

```diff
@@ -7,9 +7,13 @@ import { mock_ws } from 'Utils/mock';
 import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
 import TutorialsTabDesktop from '../tutorials-tab-desktop';
 
-jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
 jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
-jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
+
+let mockFunction: boolean | jest.Mock;
+jest.mock('lodash.debounce', () => (fn: jest.Mock) => {
+    if (!mockFunction) mockFunction = fn;
+    return mockFunction;
+});
```

Comment:
to handle this part we had to mock debounce method from lodash
<img width="558" alt="Screenshot 2024-06-24 at 18 37 28" src="https://github.com/binary-com/deriv-app/assets/103181650/55011d74-a02e-460d-8c7e-6e9385222217">